payment_term_type/README.rst

```diff
@@ -1,8 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
 Payment Terms Type
 ==================
 
-This module allow to show in payment terms the payment type depending of the lines to compute
+This module allow to show in payment terms the payment type depending of the
+lines to compute
+In Settings> Account you can find a selection field payment_type that can be
+configurated with Based on quantity of payments or Based on date of payments.
+
+Depending of the selection will affect the payment type:
 
 * Credit
+  The payment term will be credit type when the payments are covered in just
+  two ore more exhibitions (partialities) or \n when the payments are covered
+  in just one or more exhibitions, but the payments will be done in a different
+  day that the sale order confirmation day
+
 * Cash
+  The payment term will be cash type when the payments have been covered in
```

Comment:
> The payment term will be cash type **when the payments have been covered in
 just one exhibition.** without matter the date of payment or **when the payments
 have been covered in just one exhibition**, in the same day that the  sale
 order be confirmed.


This paragraph repeat the same sentence twice in he form this or that, please could you rewrote this paragh